content/en/monitors/incident_management.md

```diff
@@ -0,0 +1,101 @@
+* SEV-5: Minor issue
+* UNKNOWN: Unknown
+
+**Title**: Give your incident a descriptive title.
+
+**Signals**: The reason(s) you are declaring the incident. These can be graphs, logs, or other key visuals.
+
+**Incident commander**: This person is assigned as the leader of the incident investigation.
+
+**Additional notifications**: Notify other teams or people.
+
+Click on “Declare Incident” to finish creating your incident.
+
+### Example workflow
+
+#### 1. Discover an issue
+
+Consider a scenario in which you are looking at a dashboard, and you notice that one particular service is showing an especially high error count. Using the Export button in the upper right of a widget, you can declare an incident.
+
+{{< img src="monitors/incidents/workflow-1-graph.png" alt="From Graph"  style="width:80%;">}}
+
+#### 2. Declare an incident and assemble your team
+
+Use the New Incident modal to assemble your team and notify them. The graph from which you created the incident is automatically attached as a signal. Attach any other signals that would give your team the necessary context to begin resolving this issue. The Slack and PagerDuty integrations enable you to send notifications through those services.
+
+{{< img src="monitors/incidents/workflow-2-modal.png" alt="New Incident"  style="width:60%;">}}
+
+#### 3. Communicate and begin troubleshooting
+
+If you have the [Datadog Slack app][2] installed, the Slack integration can automatically create a new channel dedicated to the incident, so you can consolidate communication with your team and begin troubleshooting.
+
+For non-EU customers who use Slack, [sign up for beta access][3] to the Datadog Slack app. For EU customers who use Slack, stay informed about the Slack app by emailing support@datadoghq.com.
+
+{{< img src="monitors/incidents/workflow-3-slack.png" alt="Communicate"  style="width:80%;">}}
+
+#### 4. Update the incident
+
+Update the incident as the situation evolves. Set the status to `Stable` to indicate the problem has been mitigated, and set the customer impact field so that your organization knows how this issue has affected customers. Then, set the status to `Resolved` once the incident is completely fixed.
+
+There is an optional fourth status, `Completed`, that can be enabled in [Incident Settings][4].
+
+As the status of an incident changes, Datadog tracks time-to-resolution as follows:
+
+| Status Transition | Resolved Timestamp |
+| ------------------ | -----------|
+| `Active` to `Resolved`, `Active` to `Completed` | Current time |
+| `Active` to `Resolved` to `Completed`, `Active` to `Completed` to `Resolved` | Unchanged |
+| `Active` to `Completed` to `Active` to `Resolved` | Overridden on last transition |
+
+{{< img src="monitors/incidents/workflow-4-update.png" alt="Update Incident"  style="width:60%;">}}
+
+#### 5. Follow up and learn from the incident
+
+Create a postmortem with [notebooks][5], look back on exactly what went wrong, and add follow-up tasks.
+
```

Comment:
Additionally, notebooks multi-editor collaboration functionality is available in public beta. You can simultaneously edit a notebook with your response team and see who is viewing your notebook and which notebook cells they are editing.

Reply:
@niustephanie @zack-burgess

Reply:
Link to a postmortem document, look back on exactly what went wrong, and add follow-up tasks. Postmortems created in Datadog [Notebooks][5] support live collaboration; to link to an existing notebook, click the plus under `Other Docs.` Click the linked notebook to edit with teammates in real-time.